### DISCUSSION TITLE
kafka-connector has moved to new home!

### DISCUSSION THREAD

**Discussion by samarthsingal** [GoogleCloudPlatform/pubsub#325] [Announcements]
#### What's changing
The Pubsub Kafka Connector, previously live at in this repository at: https://github.com/GoogleCloudPlatform/pubsub/tree/master/kafka-connector has moved to a new home as a top-level repository at https://github.com/googleapis/java-pubsub-group-kafka-connector. 

### Source
https://github.com/GoogleCloudPlatform/pubsub/tree/master/kafka-connector will no longer receive any updates. Contributors/users are asked to change their upstreams to https://github.com/googleapis/java-pubsub-group-kafka-connector immediately.

### Releases
The new repository will be actively maintained and users can access future project releases from the new repository. Old releases will continue to be hosted here.

**Comment by madhavmadupu** [comment]
Thanks for the update! Moving the Kafka connector to its own top-level repository is a great move for modularity. It definitely makes it easier for users to track changes and manage dependencies specifically for the connector without the overhead of the main pubsub repo.
